docs/AppExtensions.md

```diff
@@ -0,0 +1,27 @@
+---
+id: app-extensions
+title: App Extensions
```

Comment:
I actually liked having App Extensions as the title, even when the Today Widget is the only one discussed so far.

If the doc is named Today Widget, I'm left wondering about how I can implement other extensions.